libs/scripts/scripts.js

```diff
@@ -41,6 +41,12 @@ const stageDomainsMap = {
     '^https://business.adobe.com/blog': 'https://main--bacom-blog--adobecom.hlx.page',
     '^https://www.adobe.com': 'origin',
   },
+  '^https://.*--milo--.*.aem.page': {
+    '^https://www.adobe.com/acrobat': 'https://main--dc--adobecom.hlx.page',
+    '^https://business.adobe.com(?!/blog)': 'https://business.stage.adobe.com',
+    '^https://business.adobe.com/blog': 'https://main--bacom-blog--adobecom.hlx.page',
```

Comment:
Why would stage not use .aem.?

Reply:
Copied it from the config above to adapt the regex!

Reply:
Hi Chris! This is just an example of how you can configure the `stageDomainsMap`. Consumers will decide if they want to convert to `.aem.` or `.hlx.` according to their specific use-cases.